[CS2103T-T13-2] Mortago

arjavibahety opened the pull request:
@dalisc
@shaoyi1997
@ambervoong
@bernicechio

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |mortuary manager |sort the bodies by certain characteristics using the `sort` command |view bodies of a speciic category and generate statistics easily
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |mortuary manager |filter the bodies by certain characteristics using `filer` command |view bodies of a certain category and generate statistics easily
```
'filer' spelling error.

tuandingwei replied:
Most of the diagrams in the Developer guide are not updated.

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |mortuary manager |filter the bodies by certain characteristics using `filer` command |view bodies of a certain category and generate statistics easily
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |mortuary manager |update the status of each and every worker, body and fridge with the `update <body/worker/fridge ID> <attribute>:<status>` command |know when (date & time) was each step of the process completed and the findings of each process (eg. cause of death)
```
Not recommended to include syntax and other implementation details (Lines 304-350).

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |mortuary manager |have a dynamically updated dashboard |reduce errors as compared to updating manually
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |mortuary manager |sort the bodies by certain characteristics using the `sort` command |view bodies of a speciic category and generate statistics easily
```
There's no prior mention of features for adding characteristics?

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |mortuary manager |add comments and feedback to workplace processes with the `feedback` command |review these feedback and improve on them
+
+|`*` |mortuary manager |feel happy when I see a beautifully designed dashboard |keep my mood up throughout the day
```
Perhaps should be a higher priority.

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`*` |mortuary manager |feel happy when I see a beautifully designed dashboard |keep my mood up throughout the day
+
+|`*` |mortuary manager |make sure that everything is organised and in order |be praised by my higher ups
```
See above.

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+
 _{More to be added}_
 
 [appendix]
```
Recommended to add additional Non-Functional Requirements and delete irrelevant ones.

ZiHawkEye commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |mortuary manager |keep track of all bodies and fridges in a single dashboard using the dashboard command |look out for any outstanding work and keep myself up to speed
 
-|`* * *` |user |add a new person |
+|`* * *` |mortuary manager |have a dynamically updated dashboard |reduce errors as compared to updating manually
```
Odd phrasing. Could be more specific since UI should be self updating by default - eg. give examples?

ZiHawkEye reviewed (commented):
Good choice of project.

Ellieyee commented on docs/DeveloperGuide.adoc:
```diff
@@ -316,14 +362,52 @@ _{More to be added}_
+
+*MSS*
+
+1. Mortuary manager requests to view the dashboard
```
This use case can be omitted.

Ellieyee commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+*Extensions*
+
+[none]
+* 3a. The list is empty.
```
This extension already falls under the MSS since there is nothing to show. 
Maybe could change it to: Application displays error indicating that there are no bodies in the list to displauy

Ellieyee commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+*Extensions*
+
+[none]
+* 1a. There are no alerts
```
Similar to the above comment

Ellieyee commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+
+1. Mortuary manager requests to undo the previous command.
+2. Mortago undoes the command.
+3. Mortago updates the GUI to reflect the new changes.
```
Could change to application displays ___

Ellieyee commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+* 2a. There is no command to undo.
++
+Use case ends.
+* 2b. An error occurred when undoing the command.
```
Could be more specific about what kind of error has occurred and how are the different ways to deal with it

Ellieyee reviewed (commented):
Could include extension or preconditions to ensure that the command format is valid. 
Could be more specific in glossary: what kind of corpse?
Overall good job :) love the idea

tuandingwei commented on docs/DeveloperGuide.adoc:
```diff
@@ -316,14 +362,52 @@ _{More to be added}_
+
+*MSS*
+
+1. Mortuary manager requests to add a body
```
Correct steps.

tohcejasmine replied:
Maybe another word instead of 'requests' can be used, since it can a little vague as to whether it involves the filling in of fields or just a request to be able to fill in the fields?

tuandingwei commented on docs/DeveloperGuide.adoc:
```diff
@@ -316,14 +362,52 @@ _{More to be added}_
+*Extensions*
+
+[none]
+* 1a. Duplicate body found.
```
You might want to define in specific what is a "duplicate".

tuandingwei commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+=== Use case: Filter entries
+*MSS*
+
+1.  Mortuary manage switches to the desired view (bodies or workers).
```
What are the workers?

tuandingwei commented on docs/DeveloperGuide.adoc:
```diff
@@ -337,10 +421,146 @@ Use case ends.
+=== Use case: Sort entries
+*MSS*
+
+1.  Mortuary manage switches to the desired view (bodies or workers).
```
What are the different desired views?

tuandingwei commented on README.adoc:
```diff
@@ -1,11 +1,11 @@
-= Address Book (Level 3)
+= Mortago
 ifdef::env-github,env-browser[:relfileprefix: docs/]
```
Well done!

tohcejasmine commented on docs/DeveloperGuide.adoc:
```diff
@@ -358,8 +578,14 @@ _{More to be added}_
-A contact detail that is not meant to be shared with others
+[[body]] Body::
+A corpse
+
```
The definitions of `fridge` and `worker` can be added in as well!

tohcejasmine reviewed (commented):
Really quirky and cool application idea! :-)

AhmadHatziq commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,62 +153,140 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
-image::UndoRedoState4.png[]
+The sequence diagram below shows how an undo command works to undo a `ClearCommand`:
+
```
Under the model, there is an additional smaller purple rectangle after "clearUndoHistory()". The purpose of this smaller rectangle is very dubious.

AhmadHatziq commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,62 +153,140 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-image::UndoRedoState5.png[]
+After this is done, if a `redo` command was executed, the `ClearCommand` would simply be executed again.
 
 The following activity diagram summarizes what happens when a user executes a new command:
```
The "if-else" conditional branch could have more specific conditions; such as "user enters valid command".

AhmadHatziq commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +315,91 @@ image::CommitActivityDiagram.png[]
+`BodyStatus`.
+
+.Notification Command Sequence Diagram
+image::NotifCommandSequenceDiagram.png[]
```
What's "up"?

Perhaps, more clearer variable names could be used.


Additionally, the source of the commands could be specified, instead of an arrow coming out of nowhere.

tuandingwei replied:
The cross should not overlap with the box. The arrows should not be pointing to the end of the box but part of the box.

AhmadHatziq commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +315,91 @@ image::CommitActivityDiagram.png[]
+
+The following sequence diagram shows how the execute operation works:
+
+image::GenReportSequenceDiagram.png[]
```
Looks well done! Clear & Concise, good job!

AhmadHatziq commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +315,91 @@ image::CommitActivityDiagram.png[]
+
+The following activity diagram summarizes what happens when a user executes a `genReport <BODY_ID>` command:
+
+image::GenReportActivityDiagram.png[]
```
Perhaps this could be more specific? 

Looks too similar with the above Activity Diagram. More differentiation might be needed.

AhmadHatziq reviewed (commented):
Good job! Don't forget to update the rest! Cheers

tuandingwei commented on docs/DeveloperGuide.adoc:
```diff
@@ -328,6 +328,71 @@ image::GenReportActivityDiagram.png[]
+The figure below illustrates the sequence diagram of the aforementioned steps.
+
+.Generation of unique `IdentificationNumber` Sequence Diagram
+image::UniqueIdentificationNumberSequenceDiagram.png[]
```
The bracket is missing in addEntity().

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update UserGuide.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

shaoyi1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

bernicechio committed:
Update bernicechio.adoc

bernicechio committed:
Reformat signature for report

bernicechio committed:
Remove all pdf

bernicechio committed:
Fix checkstyle violation

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into generate-PDF

bernicechio committed:
Fix testing

shaoyi1997 committed:
change to parsing of organs, fixes #143

ambervoong committed:
Update PPP

ambervoong committed:
Update undo/redo diagrams of DG

ambervoong committed:
Update general diagrams in the DG

ambervoong committed:
Remove .html file generated

ambervoong committed:
Merge pull request #249 from ambervoong/ambervoong-ppp

Update general diagrams in DG

shaoyi1997 committed:
add more tests and author tags, stop jumping animation when notif counter is zero

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update UserGuide.adoc

shaoyi1997 committed:
add more tests

shaoyi1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

dalisc committed:
Update DeveloperGuide.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update DeveloperGuide.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update DeveloperGuide.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update DeveloperGuide.adoc

dalisc committed:
Add files via upload

dalisc committed:
Update DeveloperGuide.adoc

dalisc committed:
Update dalisc.adoc

bernicechio committed:
Increase testing

bernicechio committed:
Remove duplicate test

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

bernicechio committed:
Merge pull request #250 from bernicechio/generate-PDF

Generate pdf v1.4

ambervoong committed:
Update PPP, UG, DG

bernicechio committed:
Update UG and bernicechio.adoc

arjavibahety committed:
Test UG

arjavibahety committed:
Edit UG

arjavibahety committed:
Edit UG

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

ambervoong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into ambervoong-ppp

ambervoong committed:
Improve code quality

ambervoong committed:
Revert "Improve code quality"

This reverts commit 029721e0b368b5a874a0260a3c83ab23897268bb.

shaoyi1997 committed:
add more tests

shaoyi1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

dalisc committed:
Merge branch 'master' into dalisc-PPP

shaoyi1997 committed:
fix tests

shaoyi1997 committed:
fix test

dalisc committed:
Update dalisc.adoc

dalisc committed:
Update dalisc.adoc

shaoyi1997 committed:
attempt to fix maximiseRestore test

shaoyi1997 committed:
separate maximise and restore test

dalisc committed:
Change sequence diagrams

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

ambervoong committed:
Test

shaoyi1997 committed:
comment out test failing in travis

ambervoong committed:
trailing white space

shaoyi1997 committed:
fix checkstyle

dalisc committed:
Merge pull request #256 from dalisc/dalisc-PPP

Update PPP, UG, DG

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

ambervoong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset4

shaoyi1997 committed:
fix performanceTest for workerlistpaneltest

shaoyi1997 committed:
Merge pull request #255 from shaoyi1997/master

Add more Tests and fix maximisation

ambervoong committed:
Merge pull request #251 from ambervoong/ambervoong-ppp

Update PPP, UG, DG

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

bernicechio committed:
Update UG, DG

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

bernicechio committed:
Update bernicechio.adoc

bernicechio committed:
Merge pull request #253 from bernicechio/bernice-PPP

Update PPP, UG, DG

bernicechio committed:
Update UG, PPP

bernicechio committed:
Update UG

arjavibahety committed:
Change UG and DG

arjavibahety committed:
Delete queued runnables

ambervoong committed:
Fix UML diagram
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset5

bernicechio committed:
Update gh-pages.css

ambervoong committed:
Merge pull request #259 from ambervoong/test_reset5

Change sequence diagram, documentation changes

shaoyi1997 committed:
UG changes and selected body bg color change

shaoyi1997 committed:
fix performanceTest for fridgeGridViewTest

ambervoong committed:
Change error messages/output messages

shaoyi1997 committed:
Merge branch 'master' into master

shaoyi1997 committed:
Merge pull request #261 from shaoyi1997/master

Updates to Ui and UG

dalisc committed:
Add new diagrams

ambervoong committed:
Fix checkstyle

dalisc committed:
Checkstyle

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

dalisc committed:
Merge pull request #263 from dalisc/ui-instruction-diagrams

Ui instruction diagrams

ambervoong committed:
Diagram colour change

bernicechio committed:
Update *.adoc and changed message usage for generate pdf

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

ambervoong committed:
Merge pull request #262 from ambervoong/test_reset5

Change error messages/output messages

bernicechio committed:
Disallow signature to contain special characters

bernicechio committed:
Fix bug

dalisc committed:
Add captions to stats chart

bernicechio committed:
Increase testing

bernicechio committed:
Update UG

dalisc committed:
Increased resolution for diagrams

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

bernicechio committed:
Add screenshots to UG

dalisc committed:
Merge pull request #264 from dalisc/ui-instruction-diagrams

Increased resolution for diagrams

shaoyi1997 committed:
edit UG

bernicechio committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into bernice-PPP

shaoyi1997 committed:
Merge pull request #265 from shaoyi1997/master

Edit UG and Refactoring of FridgeIdLabel

bernicechio committed:
Merge branch 'master' into bernice-PPP

bernicechio committed:
Update UG and bernicechio.adoc

bernicechio committed:
Merge branch 'bernice-PPP' of https://github.com/bernicechio/main into bernice-PPP

bernicechio committed:
Merge pull request #260 from bernicechio/bernice-PPP

Update UG and disallow special characters for signatures

dalisc committed:
Change list diagram

dalisc committed:
Merge pull request #266 from dalisc/ui-instruction-diagrams

Change list diagram

bernicechio committed:
Change color to purple

arjavibahety committed:
Fix errors in notif and update parser

arjavibahety committed:
Merge branch 'master' into master

arjavibahety committed:
Edit DG

arjavibahety committed:
Merge branch 'master' of https://github.com/arjavibahety/main

ambervoong committed:
Add image to UG

ambervoong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset5

arjavibahety committed:
Fix EOF

arjavibahety committed:
Fix UG merge conflicts

arjavibahety committed:
Merge pull request #252 from arjavibahety/master

Edit UG, fix notif command and update command parser

ambervoong committed:
Update diagrams, add screenshots to UG

arjavibahety committed:
Change UG diagrams

arjavibahety committed:
Empty fridge when body is donated

arjavibahety committed:
Update UG

ambervoong committed:
Update UG DG, Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset5

# Conflicts:
#	docs/UserGuide.adoc

ambervoong committed:
Update UGDG formatting

arjavibahety committed:
Fix UG

ambervoong committed:
Merge pull request #268 from ambervoong/test_reset5

Update UG, DG diagrams and formatting. Update PPP

arjavibahety committed:
Edit DG

arjavibahety committed:
Merge pull request #267 from arjavibahety/master

Update UG, DG, and empty fridge when body is donated

ambervoong committed:
Attempt to change gh pages.css, optimise PPP for printing

ambervoong committed:
Merge pull request #269 from ambervoong/test_reset5

Attempt to change gh pages.css, optimise PPP for printing

dalisc committed:
Update PPP, UG, DG

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into documentation

dalisc committed:
Fix filter for status bug

dalisc committed:
Merge pull request #270 from dalisc/documentation

Documentation

dalisc committed:
Fixed issues #191, #192, #193

dalisc committed:
Fix checkstyle

bernicechio committed:
Repush

bernicechio committed:
Merge branch 'master' into bernice-PPP

bernicechio committed:
Merge pull request #272 from bernicechio/bernice-PPP

Update documentation

bernicechio committed:
Fix screenshots

bernicechio committed:
Fix screenshots

bernicechio committed:
Fix screenshots

bernicechio committed:
Merge pull request #273 from bernicechio/bernice-PPP

Fix screenshots for UG

shaoyi1997 committed:
fixes #209, closes #183

bernicechio committed:
Update invalid signature message

bernicechio committed:
Merge pull request #274 from bernicechio/bernice-PPP

Update invalid signature message

bernicechio committed:
Cut PPP to limit 10 pages max

bernicechio committed:
Merge pull request #275 from bernicechio/bernice-PPP

Cut PPP to limit 10 pages max

ambervoong committed:
Change shadowJar name

shaoyi1997 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main

ambervoong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset5

bernicechio committed:
Remove purple markup

bernicechio committed:
Merge pull request #277 from bernicechio/bernice-PPP

Remove purple markup

shaoyi1997 committed:
update to UGDG

ambervoong committed:
Move unused code, fix update invalid dates for Worker

ambervoong committed:
Fix eof

ambervoong committed:
Fix invalid dates for update

ambervoong committed:
Implement date validity checking in Update.

arjavibahety committed:
Edit documentation

ambervoong committed:
Merge pull request #276 from ambervoong/test_reset5

Add validity checking for dates in Updatecommand

ambervoong committed:
Remove purple markup from UG

dalisc committed:
Pass tests

dalisc committed:
Merge branch 'master' into feature-filter-command

dalisc committed:
Merge pull request #271 from dalisc/feature-filter-command

Fix filter for status bug

dalisc committed:
Replace ) with . for captions

dalisc committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into documentation

dalisc committed:
Update PPP and DG

bernicechio committed:
Update DeveloperGuide.adoc

dalisc committed:
Removed line chart panel test

arjavibahety committed:
Edit docs

dalisc committed:
Fix checkstyle

ambervoong committed:
Change product name

dalisc committed:
Edit DG

dalisc committed:
Edit UG

dalisc committed:
Fix checkstyle

dalisc committed:
Edit UG

dalisc committed:
Edit PPP

dalisc committed:
Edit PPP

dalisc committed:
Edit PPP

ambervoong committed:
Update manual testing, change product name in various .adoc files.

arjavibahety committed:
Edit UG DG PPP

ambervoong committed:
Merge branch 'master' into test_reset5

dalisc committed:
Remove asciidoctor generated HTML

ambervoong committed:
Merge pull request #280 from ambervoong/test_reset5

Update links in various .adoc files to reflect Mortago, add manual testing

arjavibahety committed:
Fix issue with images

arjavibahety committed:
Remove html files

arjavibahety committed:
Merge pull request #279 from arjavibahety/master

Edit documentation

arjavibahety committed:
Fix error when updating a claimed or donated body

arjavibahety committed:
Merge pull request #281 from arjavibahety/master

Fix error when updating a claimed or donated body

ambervoong committed:
Update PPP

shaoyi1997 committed:
edited UGDG and PPP

ambervoong committed:
Fix typo

shaoyi1997 committed:
Merge branch 'master' into master

ambervoong committed:
Merge branch 'test_reset5' of https://github.com/ambervoong/main into test_reset5

# Conflicts:
#	docs/DeveloperGuide.adoc

ambervoong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T13-2/main into test_reset5

ambervoong committed:
Merge pull request #283 from ambervoong/test_reset5

Fix typo

arjavibahety committed:
Edit docs

shaoyi1997 committed:
fix test fail

shaoyi1997 committed:
update ppp

shaoyi1997 committed:
resolve conflict

shaoyi1997 committed:
Merge branch 'master' of https://github.com/shaoyi1997/main

arjavibahety committed:
Update ppp

dalisc committed:
Update UserGuide.adoc

dalisc committed:
Merge branch 'master' into documentation

dalisc committed:
Merge pull request #278 from dalisc/documentation

Documentation

arjavibahety committed:
Edit docs

arjavibahety committed:
Merge branch 'master' into master

arjavibahety committed:
Merge pull request #284 from arjavibahety/master

Edit docs

shaoyi1997 committed:
comment out erroneous block

arjavibahety committed:
Update UserGuide.adoc

shaoyi1997 committed:
Merge branch 'master' into master

arjavibahety committed:
Update UserGuide.adoc

shaoyi1997 committed:
fix checkstyle

shaoyi1997 committed:
Merge branch 'master' of https://github.com/shaoyi1997/main

shaoyi1997 committed:
Merge pull request #282 from shaoyi1997/master

Edit UGDG, PPP and some bug fixes

shaoyi1997 committed:
Update DeveloperGuide.adoc

arjavibahety committed:
Remove sizing

arjavibahety committed:
Merge branch 'master' of https://github.com/arjavibahety/main

arjavibahety committed:
Merge pull request #285 from arjavibahety/master

Remove sizing

arjavibahety committed:
Add manual testing and fix consistency issues

arjavibahety committed:
Merge pull request #286 from arjavibahety/master

Add manual testing and fix consistency issues

shaoyi1997 committed:
update PPP, UGDG

ambervoong committed:
Fix typos in UG

shaoyi1997 committed:
resolve conflict

shaoyi1997 committed:
resolve conflict

shaoyi1997 committed:
Merge pull request #288 from shaoyi1997/master

Update PPP, UGDG

ambervoong committed:
Merge pull request #287 from ambervoong/test_reset5

Fix typos in docs

ambervoong committed:
Fix DG image numbering

ambervoong committed:
Merge pull request #289 from ambervoong/test_reset5

Fix DG image numbering

shaoyi1997 committed:
change Readme

shaoyi1997 committed:
Merge pull request #290 from shaoyi1997/master

change Readme

ambervoong committed:
Delete image
